webcompat/static/css/development/components/comment.css

```diff
@@ -161,7 +166,6 @@
 
   /* QR */
   .wc-Comment-qr {
-    position:absolute;
     z-index: 1;
-    right: 0;
+    margin: -6px 16px 0 auto;
```

Comment:
@dshgna if you  add `justify-content: space-between` in `wc-Comment-header` you don't need margin